### PR TITLE
Basic accordion implementation

### LINKED ISSUE #30: [Blocks] Implement accordion component
Sample pages

https://docs.google.com/document/d/1Keaa7phnBrZh1qRoAphTU2yxAnGv-NMx0LnmSyS3cfc/edit
https://docs.google.com/document/d/1ceaXivZOPSSCQSoRotFpK6pRJ26VBuZkr3fkSSJ82BU/edit

- [ ] clarify with authors which variants exist

### PR DESCRIPTION
**Pull request by alexcarol**
Implements the accordion component

What's included:
- Sections with the "accordion" class will behave as such (only one element open at a time)
- Transitions
- Title text size

What's not included:
- Text color (the original site uses an "accordion-title-dark-blue" class, should we just default to blue, or is there any better approach)
- Original site flex behaviour (it should probably depend on the page)
- Media queries for "open accordion item" icon size (is this a desirable behaviour?)

Note: if you have multiple groups of "accordions" in the same page only one accordion item across the whole page can be open at a time. This can be changed, but i'd rather discuss whether the current behaviour is a problem at all before making any further changes.

Fix #30 

Test URLs:
- Original: https://www.netcentric.biz/what-we-do/digital-strategy
- Before: https://main--netcentric--hlxsites.hlx.page/what-we-do/digital-strategy
- After: https://issue-30--netcentric--hlxsites.hlx.page/what-we-do/digital-strategy

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -27,6 +27,48 @@ function buildHeroBlock(main) {
...
 }
 
+function createIcon(name) {
+  const icon = document.createElement('span');
+  icon.classList.add('icon', `icon-${name}`);
+
+  return icon;
+}
+
+function buildAccordionBlock(main) {
```
Comment (mhaack) [inline scripts/scripts.js:37]: That method should go into the `accordion.js`

Comment (alexcarol) [reply scripts/scripts.js]: Fixed

**Review by hannessolo — CHANGES_REQUESTED** [review]

**Comment by hannessolo** [comment]
Making it accessible shouldn't take much, if you use `aria-expanded` instead of `.open` you are already half way there, http://web-accessibility.carnegiemuseums.org/code/accordions/ here are the guidelines.

File: scripts/scripts.js [hunk 2]
```diff
@@ -135,6 +136,7 @@ export function decorateTwitterFeed(main) {
 function buildAutoBlocks(main) {
   try {
     buildHeroBlock(main);
+    buildAccordionBlock(main);
```
[the remark this answers is not in the document]
Comment (alexcarol) [reply scripts/scripts.js]: Thanks, fixed!

File: scripts/lib-franklin.js [hunk 3]
```diff
@@ -123,6 +123,18 @@ export function toCamelCase(name) {
   return toClassName(name).replace(/-([a-z])/g, (g) => g[1].toUpperCase());
 }
 
+/**
+ * Returns a Franklin icon span (that will be expanded by decorateIcons)
+ * @param {string} name The icon file name (minus ".svg")
+ * @returns {HTMLSpanElement}
+ */
+export function createIcon(name) {
```
Comment (hannessolo) [inline scripts/lib-franklin.js:131]: AFAIK global custom code should not be placed in `lib-franklin.js` but rather in `scripts.js`, as `lib-franklin` is basically the versioned framework code.

Comment (alexcarol) [reply scripts/lib-franklin.js]: Fixed

**Comment by alexcarol** [comment]
> Making it accessible shouldn't take much, if you use `aria-expanded` instead of `.open` you are already half way there, http://web-accessibility.carnegiemuseums.org/code/accordions/ here are the guidelines.

Thanks for the feedback, it has been addressed.

**Review by hannessolo — APPROVED** [review]
Looks good now!

**Pull request merged**